Client/src/Components/DataCell/DataCell.js

```diff
@@ -1,47 +1,89 @@
-import Box from '@material-ui/core/Box'
-import Button from '@material-ui/core/Button'
-import { Container } from '@material-ui/core'
-import React from 'react'
-
-/**
- * This component handles receiving the json locally then sending the file for processing
- * to the correct API.
- */
-export default function DataCell() {
-  /**
-     * Upon submission, the JSON file is extracted from the event and must be appended to formData
-     * to be sent with API request.
-     */
-  const handleSubmit = async (e) => {
-    e.preventDefault()
-
-    const json = e.target.file.files[0]
-
-    const formData = new FormData()
-    formData.append('file', json)
-
-    const options = {
-      method: 'POST',
-      body: formData,
-    }
-    await fetch('http://localhost:4000/dataupload', options)
-      .then(resp => resp.json())
-      .then(result => {
-        console.log(result[0])
-      })
-  }
-
-  return (
-    <Container>
-      <Box border={30} p={4} borderColor="primary">
-        <form onSubmit={handleSubmit}>
-          <img src={require('./uploadimage.png')} alt="Visual of clouds"></img>
-          <div>
-            <input type="file" id="file" accept=".json" />
-            <Button type="submit" variant="contained"> Upload this file! </Button>
-          </div>
-        </form>
-      </Box>
-    </Container>
-  )
-}
+//Verify that material-ui lab and material-ui core are installed before running this file!
+// npm install @material-ui/lab

```

Comment:
again this should not be here, 'npm install' installs all the packages inside the package.json in the case they are not installed already, we do not need to leave comments about this. Just run 'npm install' every time you pull develop

Reply:
consult the docs: 
![image](https://user-images.githubusercontent.com/22376976/101484107-1e500300-3927-11eb-8317-910112d13e3c.png)